src/mlpack/core/optimizers/problems/spsa_test_function.cpp

```diff
@@ -0,0 +1,40 @@
+/**
+ * @file sgd_test_function.cpp
```

Comment:
I forgot to change the comments part... 😅 I created a new test function because I wanted `Evaluate()` to return a vector whereas `Evaluate()` from `SGDTestFunction` returns only a single value.

Reply:
@zoq I used `SPSATestFunction` to implement a `Evaluate()` function which can process and return a vector which was not available in `SGDTestFunction`. What do you think?